Gopkg.toml

```diff
@@ -1,10 +1,24 @@
 required = [
+	"github.com/getlantern/deepcopy",
 	"github.com/ghodss/yaml",
-	"github.com/openshift/ci-operator/pkg/api",
 	"github.com/sirupsen/logrus",
-	"k8s.io/test-infra/prow/config",
+
+	"github.com/openshift/ci-operator/pkg/api",
+
+	"k8s.io/test-infra/prow/apis/prowjobs/v1",
+	"k8s.io/test-infra/prow/client/clientset/versioned",
+	"k8s.io/test-infra/prow/client/clientset/versioned/typed/prowjobs/v1",
+
 	"k8s.io/test-infra/prow/kube",
+	"k8s.io/test-infra/prow/config",
+	"k8s.io/test-infra/prow/pjutil",
+
+	"k8s.io/client-go/tools/clientcmd",
+	"k8s.io/client-go/kubernetes/typed/core/v1",
```

Comment:
Why didn't this fall out in the rebase?

Reply:
Not sure I understand the question. We use this.

Reply:
I thought you put in the deps first, though?

Reply:
The first (already merged) PR only updated what we already vendored for Prowgen itself, mostly `test-infra` stuff and its deps. Looks like these were previously only pulled transitively and were added to `Gopkg.toml` because rehearse use these directly.